craco.config.js

```
@@ -7,21 +7,41 @@ module.exports = {
+          stream: require.resolve('stream-browserify'),
+          buffer: require.resolve('buffer/'),
+          process: require.resolve('process/browser.js'),
         },
-      },
-      plugins: [
+      };
+
+      config.module = {
+        ...config.module,
+        rules: config.module.rules.map((rule) => {
+          if (rule.oneOf instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)